lib/streamlit/runtime/state/session_state.py

```diff
@@ -369,8 +414,7 @@ def filtered_state(self) -> dict[str, Any]:
     @property
     def _reverse_key_wid_map(self) -> dict[str, str]:
```

Comment:
Do we even still need this property? Can't we just use `self._key_id_mapper._id_key_mapping` everywhere where `_reverse_key_wid_map` is used?

Reply:
Good point, I removed this function